roles/ceph-common/tasks/checks/check_ntp_atomic.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: check ntp installation on atomic
+  command: rpm -q chrony
+  register: ntp_pkg_query
+  ignore_errors: true
+  changed_when: false
+  when: is_atomic
```

Comment:
the `when` is not needed here since there is already one on the `include` below.